net/frp/files/frpc.init

```diff
@@ -35,6 +37,22 @@ config_cb() {
 	fi
 }
 
+upgrade() {
```

Comment:
This should happen in a UCI default script. Otherwise it is always done at the startup of the service!

Reply:
Done